velox/connectors/hive/tests/HiveDataSinkTest.cpp

```diff
@@ -643,14 +708,14 @@ TEST_F(HiveDataSinkTest, memoryReclaim) {
           expectedWriterReclaimed);
     }
   } testSettings[] = {
-      //    {dwio::common::FileFormat::DWRF, true, true, 1 << 30, true, true},
-      {dwio::common::FileFormat::DWRF, true, true, 1, true, true},
-      {dwio::common::FileFormat::DWRF, true, false, 1 << 30, false, false},
-      {dwio::common::FileFormat::DWRF, true, false, 1, false, false},
-      {dwio::common::FileFormat::DWRF, false, true, 1 << 30, true, false},
-      {dwio::common::FileFormat::DWRF, false, true, 1, true, true},
-      {dwio::common::FileFormat::DWRF, false, false, 1 << 30, false, false},
-      {dwio::common::FileFormat::DWRF, false, false, 1, false, false},
+    //    {dwio::common::FileFormat::DWRF, true, true, 1 << 30, true, true},
```

Comment:
Why is this commented out? Let's remove.

Reply:
@mbasmanova Removed.